hs_core/templates/pages/genericresource.html

```diff
@@ -329,8 +329,8 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                 </div>
                                 <div class="modal-body">
                                     <div class="warnings-list">
-                                        <div>Consider saving a copy of the files before you delete it if it is
```

Comment:
Is this messaging supposed to be changed as well?

Reply:
Confirmed via email with Jeff. This is the appropriate text.